tests/helpers/test_pytorch_model.py

```diff
@@ -157,3 +150,28 @@ def test_get_random_layer_generator(load_mnist_model):
         new_layer = getattr(random_layer_model, layer_name).parameters()
 
         assert layer != new_layer, "Test failed."
+
+
+@pytest.mark.pytorch_model
+@pytest.mark.parametrize(
+    "params", [
+        {},
+        {'layer_names': ['conv_2']},
+        {'layer_indices': [0, 1]}
+    ],
+    ids=[
+        'all layers',
+        '2nd conv',
+        '1st 2 layers'
```

Comment:
Please go back to get_hidden_layers and explain the options (str) clearly in the docstring!

Reply:
done https://github.com/aaarrti/Quantus/blob/relative-stability/quantus/helpers/model_interface.py#L53-L68

Reply:
done here https://github.com/aaarrti/Quantus/blob/relative-stability/quantus/helpers/model_interface.py#L53-L68

Reply:
100%!